scripts/errors-patch/patches/009-stderrors.patch

```diff
@@ -0,0 +1,26 @@
+@@
+var errors identifier
+@@
+import errors "github.com/juju/errors"
++ import interrors "github.com/juju/juju/internal/errors"
```

Comment:
Can we please, please avoid abbreviations here :-)
We already use `internalerrors` in approx 30 places. Can we standardise on that.

The other option is to recognise that we'd be importing internal errors way more often than "errors" so we should just leave internal errors as "errors" and alias "errors" to "stderrors"

Reply:
these are temporary renames.

Reply:
Temp renames they're fixed up later in patches. You have to  do everything as small steps.